page_creation.rst

```diff
@@ -209,11 +209,11 @@ Bundles are registered in your ``app/AppKernel.php`` file (a rare PHP file in th
     {
         public function registerBundles()
         {
-            $bundles = array(
+            $bundles = [
```

Comment:
@xabbuh Could we keep the new array syntax here? We already used it in [Symfony SE v3](https://github.com/symfony/symfony-standard/blob/3.0/app/AppKernel.php#L10-L19).